drivers/display/ssd1306.c

```diff
@@ -434,7 +434,8 @@ static int ssd1306_init(struct device *dev)
 
 	gpio_pin_configure(driver->reset,
 			   DT_INST_0_SOLOMON_SSD1306FB_RESET_GPIOS_PIN,
-			   GPIO_DIR_OUT);
+			   GPIO_OUTPUT_HIGH |
```

Comment:
We want the driver to honor `GPIO_ACTIVE_LOW`, `GPIO_ACTIVE_HIGH` flags, therefore we should use `GPIO_OUTPUT_INACTIVE` here.

Reply:
fixed